Remove fake MIME types that cause warnings.

The offending MIME types are commented out rather than
physically removed because doing so would cause the patch
to be too large.

diff --git a/mimetypes/kde.xml b/mimetypes/kde.xml
index 76303db..23351c4 100644
--- a/mimetypes/kde.xml
+++ b/mimetypes/kde.xml
@@ -1783,8 +1783,8 @@ Notes:
     </magic>
     <glob pattern="*.abc"/>
   </mime-type>
-
-  <!-- all/ fake mime types -->
+<!--
+  all/ fake mime types
   <mime-type type="all/all">
     <comment>all files and folders</comment>
     <comment xml:lang="ar">كل الملفات و المجلدات</comment>
@@ -1902,7 +1902,7 @@ Notes:
     <comment xml:lang="zh_TW">所有檔案</comment>
   </mime-type>
 
-  <!-- uri/ fake mime types -->
+  uri/ fake mime types
   <mime-type type="uri/mms">
     <comment>mms: URIs</comment>
     <comment xml:lang="ar">mms: وصلات</comment>
@@ -2245,7 +2245,7 @@ Notes:
     <comment xml:lang="zh_CN">rtspu: 地址</comment>
     <comment xml:lang="zh_TW">rtspu: 網址</comment>
   </mime-type>
-
+-->
   <mime-type type="application/vnd.kde.fontspackage">
     <sub-class-of type="application/zip"/>
     <comment>fonts package</comment>
